src/file_list/select.js

```diff
@@ -12,10 +12,8 @@ export default class Select {
   }
 
   indexOf (item) {
-    var children = this.items()
-    var i = 0
-    for (; i < children.length; i++) {
-      if (children[i] === item) {
+    for (var i = 0; i < this.items().length; i++) {
+      if (this.items()[i] === item) {
```

Comment:
This is not good practice, keep children var to not call this every single iteration. What happens if items() suddenly changes? Can cause infinite loop.